api/v1/playbook_types.go

```diff
@@ -142,6 +142,8 @@ type PlaybookTrigger struct {
 }
 
 type PlaybookSpec struct {
+	Title string `json:"title,omitempty" yaml:"title,omitempty"`
```

Comment:
Did we not want to add category as well ?

Reply:
👍🏽